ADR/ADR022-use-postgresql-for-form-creation-in-form-admin.md

```diff
@@ -0,0 +1,41 @@
+# ADR022:  (a description of the decision, not the problem)
+
+Date: 2023-09-14
+
+## Status
+
+> Proposed
+
+## Context
+
+Originally forms-admin didn't have a workflow but a single screen for the user to draft their form. This got broken up into a multistage workflow. Data is transferred between screens via cookies but this has limitations, and only worked ok for a linear flow. Now that we are approaching diverging workflows, we can't support this any longer. We are also currently only using ActiveModel for our classes.
+
+We should:
+
+* Refactor our models to be more Rails-standard and switch from ActiveModel only to ActiveRecord.
+
+* Use the PostgreSQL database to manage data.
```

Comment:
Is this a standard Rails thing? Is there documentation on how to do this we can refer to?

Reply:
We can either store sessions in db, or migrate from that (later!) and only have session for logged in user and move to a more author per form version model